haas/ext/switches/brocade.py

```diff
@@ -277,8 +278,11 @@ def _construct_tag(name):
         """ Construct the xml tag by prepending the brocade tag prefix. """
         return '{urn:brocade.com:mgmt:brocade-interface}%s' % name
 
-    def _make_request(self, method, url, data=None):
+    def _make_request(self, method, url, data=None,
+                      acceptable_error_codes=None):
         r = requests.request(method, url, data=data, auth=self._auth)
-        if r.status_code >= 400:
+        acceptable_error_codes = acceptable_error_codes or []
```

Comment:
Instead of doing this, in the signature you can just define the default as `()` (empty tuple). Note that you should *not* define the default as `[]`, as lists are mutable and if we ever change this function to modify the argument, those modifications will propogate to future calls.